build/build-bundle.js

```diff
@@ -130,7 +130,7 @@ async function build(entryPath, distPath, opts = {minify: true}) {
           '__filename': (id) => `'${path.relative(LH_ROOT, id)}'`,
           // This package exports to default in a way that causes Rollup to get confused,
           // resulting in MessageFormat being undefined.
-          'require(\'intl-messageformat\').default': 'require(\'intl-messageformat\')',
+          // 'require(\'intl-messageformat\').default': 'require(\'intl-messageformat\')',
```

Comment:
deferring to GHA to verify this, having unrelated trouble with my mac running devtools locally